.gitignore

```diff
@@ -21,4 +21,7 @@ bin/
 # Logs
 logs/
 **.log
-simulation.json
\ No newline at end of file
+simulation.json
+
+# Frontend
+**/build
```

Comment:
GitHub complaining about no newline at end of file so might as well change it I guess (or not... I don't really care)